react/components/MiniCartContent.js

```diff
@@ -164,6 +164,7 @@ class MiniCartContent extends Component {
 
   createProductShapeFromOption = (option) => ({
     ...this.createProductShapeFromItem(option),
+    compositionItem: getCompositionForOption(option, this.props.data.orderForm),
```

Comment:
:pick: perhaps `getOptionComposition`, to be in line with `getOptionChoiceType`

Reply:
will do